features/TESTS/filesystem/util_block_device/main.cpp

```diff
@@ -0,0 +1,171 @@
+#include "mbed.h"
```

Comment:
I've been using `TEST_ASSERT_EQUAL` since it will log the values it compared on failure, giving the user the return code of the function and line of the assert. I can also use `TEST_ASSERT_EQUAL_MESSAGE` to give context, I'll see where I can adopt this.